WordPress/Classes/Services/AccountService.h

```diff
@@ -86,6 +86,12 @@ extern NSString *const WPAccountEmailAndDefaultBlogUpdatedNotification;
 - (void)updateUserDetailsForAccount:(WPAccount *)account success:(void (^)())success failure:(void (^)(NSError *error))failure;
 
 /**
+ Initializes the WordPress iOS Extensions with the WordPress.com Default Account.
+ */
+- (void)setupExtensionsWithDefaultAccount;
```

Comment:
can we call this `setupAppExtensionsWithDefaultAccount` so it's super clear it's not class/Swift extensions ... I know I know...